composer.json

```
@@ -20,6 +20,9 @@
         "symfony/dom-crawler": "^3.4|^4.4|^5.4|^6.0"
     },
     "require-dev": {
+        "dealerdirect/phpcodesniffer-composer-installer": "^1.0",
+        "phpcompatibility/php-compatibility": "^9.3",
+        "phpcsstandards/php_codesniffer": "^3.7",
```

```suggestion
        "squizlabs/php_codesniffer": "^3.7",
```

Thank you for your support and your enthousiasm embracing the take-over of the PHP_CodeSniffer package.

In contrast to earlier information, arrangements are now being made to allow the package to continue under its original name on Packagist. The commit (in the new repo) to rename the package [has been reverted](https://github.com/PHPCSStandards/PHP_CodeSniffer/pull/135).

Sorry for the confusion and thank you for understanding. I hope you'll enjoy the 3.8.0 release, which I expect to release this Friday.